.github/workflows/main.yml

```diff
@@ -20,57 +20,92 @@ jobs:
             task: r-package
             compiler: gcc
             r_version: 3.6
+            build_type: cmake
           - os: ubuntu-latest
             task: r-package
             compiler: gcc
             r_version: 4.0
+            build_type: cmake
           - os: ubuntu-latest
             task: r-package
             compiler: clang
             r_version: 3.6
+            build_type: cmake
           - os: ubuntu-latest
             task: r-package
             compiler: clang
             r_version: 4.0
+            build_type: cmake
           - os: macOS-latest
             task: r-package
             compiler: gcc
             r_version: 3.6
+            build_type: cmake
           - os: macOS-latest
             task: r-package
             compiler: gcc
             r_version: 4.0
+            build_type: cmake
           - os: macOS-latest
             task: r-package
             compiler: clang
             r_version: 3.6
+            build_type: cmake
           - os: macOS-latest
             task: r-package
             compiler: clang
             r_version: 4.0
+            build_type: cmake
           - os: windows-latest
             task: r-package
             compiler: MINGW
             toolchain: MINGW
             r_version: 3.6
+            build_type: cmake
           - os: windows-latest
             task: r-package
             compiler: MINGW
             toolchain: MSYS
             r_version: 4.0
+            build_type: cmake
           # Visual Studio 2017
           - os: windows-2016
             task: r-package
             compiler: MSVC
             toolchain: MSVC
             r_version: 3.6
+            build_type: cmake
           # Visual Studio 2019
           - os: windows-2019
             task: r-package
             compiler: MSVC
             toolchain: MSVC
             r_version: 4.0
+            build_type: cmake
+          ###############
+          # CRAN builds #
+          ###############
+          - os: windows-latest
+            task: r-package
+            compiler: MINGW
+            toolchain: MSYS
```

Comment:
having them makes naming the jobs easier. Otherwise I'd have to add a `subtitle` or something that is a hard-coded string for each job so we can tell them apart visually in the GitHub Actions UI. Do you think that would be better?

Reply:
Ah, I forget about that! Of course, the current code is better.